lib/ExpensiMark.js

```diff
@@ -49,6 +49,29 @@ export default class ExpensiMark {
                 replacement: '$1<code>$2</code>$3',
             },
 
+            /**
+             *Apply the hereMention first because the string @here is still a valid mention for the userMention regex.
+             *This ensures that the hereMention is always considered first, even if it is followed by a valid userMention.
+            */
+            {
+                name: 'hereMention',
+                regex: /(^|\s)@here\b/g,
+                replacement: match => `<mention-here>${match}</mention-here>`,
+            },
+
+            /**
+             * This regex matches a valid user mention in a string.
+             * A user mention is a string that starts with the '@' symbol and is followed by a valid username.
```

Comment:
For now, actually, a user mention is @ followed by primary login. And primary logins area always in the form of email addresses. For example: 
- puneet@expensify.com
- +19726585678@expensify.sms

So I think we can make this a little less permissive.

Reply:
yes, that sounds good!